### ISSUE TITLE
More keyboard shortcuts

### ISSUE THREAD

**Issue by george9816** [roldanjr/pomatez#325]
I'm always frustrated when I need to open the app for stopping the timer, instead of using a shortcut.

Please add shortcuts for starting / pausing the timer. :)

**Comment by sekwah41** [comment]
Are you referring to global shortcuts? Do you have any in mind?

**Comment by george9816** [comment]
I'm reffering to global shortcuts, yes.
What are you reffering at with „do you have any in mind?”. What I wish are customisable global shortcuts for start/pause the timer. :) I think they would be very useful.

**Comment by sekwah41** [comment]
I mean as in are there any suggested key combos youd like?

**Comment by george9816** [comment]
No, any would be good. As for an example, Ctrl+F1 can be used, as it also is in Pomotroid.

**Comment by Cheve11e** [comment]
Yes, I can agree on this. Keyboard shortcuts for Start/Pause timer are needed. Please add this :)
